api/api_errors.go

```diff
@@ -4,9 +4,27 @@ import (
 	"errors"
 	"reflect"
 
+	"golang.org/x/xerrors"
+
 	"github.com/filecoin-project/go-jsonrpc"
 )
 
+var invalidExecutionRevertedWithDataMsg = xerrors.New("invalid execution reverted with data error")
+
+// Default messages for errors.
+const (
```

Comment:
I don't think these are going to be appreciated, there's already 3 places that some of the errors in this file appear or have constants, it's probably overkill to pull these strings up to constants.